pilz_teleoperation/test/integrationtests/test_key_teleop.py

```diff
@@ -31,24 +31,39 @@ def __init__(self, *args, **kwargs):
 
     def setUp(self):
         rospy.loginfo("SetUp called...")
+        self.twist_published = False
 
     def tearDown(self):
         rospy.loginfo("TearDown called...")
 
     def callback(self, msg):
         self.twist_published = True
 
-    def test_startup(self):
+    def test_startup_twist(self):
         """ Test if driver gets started """
-        subscriber_twist = rospy.Subscriber('/jog_server/delta_jog_cmds', TwistStamped, self.callback)
-        subscriber_joint = rospy.Subscriber('/jog_server/joint_delta_jog_cmds', JointJog, self.callback)
+        rospy.Subscriber('/jog_server/delta_jog_cmds', TwistStamped, self.callback)
+        self._publish_object_to_topic_and_assert(what=Twist(linear=Vector3(y=1)),
+                                                 topic='/key_input/twist')
+
+    def _publish_object_to_topic_and_assert(self, what, topic):
+        publisher_twist = rospy.Publisher(topic, type(what), queue_size=1)
+        self._wait_for_event_with_timeout(lambda: publisher_twist.get_num_connections() < 1,
+                                          secs=5,
+                                          msg="subscriber did not come up")
+        publisher_twist.publish(what)
+        self._wait_for_event_with_timeout(lambda: self.twist_published, secs=5, msg="driver did not publish")
```

Comment:
self.twist_published is also used by the startup_jog test, thus the name obscures the variable's use. I recommend renaming this variable to object_is_published.